File: LeetCode/Easy/1598-crawler-log-folder/1598-crawler-log-folder.py
class Solution:
    def minOperations(self, logs: List[str]) -> int:
        ret = 0
        
        for log in logs:
            if log[0] == ".":
                if log[1] == ".":
                    ret = max(0, ret - 1)
            else:
                ret += 1
        
        return ret

File: LeetCode/Easy/1598-crawler-log-folder/1598-crawler-log-folder.go
func minOperations(logs []string) int {
    ret := 0
        
    for _, log := range logs {
        if log[0] == '.' {
            if log[1] == '.' {
                ret = max(0, ret - 1)
            }
        } else { ret++ }
    }

    return ret
}


func max (a, b int) int {
    if a > b { return a }
    return b
}